docs/faq.md

```diff
@@ -152,6 +152,23 @@ In either case, look at the
 [coverage reports]({{ site.baseurl }}/further-reading/clusterfuzz#coverage-reports)
 for your target(s) and figure out why some parts of the code are not covered. 
 
+## What if my fuzzer does not find new coverage or bugs after a while?
+
+It is common that the fuzzer gets saturated and cannot find new coverage or bugs after a while.
+We have designed and developed [Fuzz Introspector](https://github.com/ossf/fuzz-introspector)
+to give developers a tool to evaluate the fuzzers performance. Fuzz Introspector aims to improve 
+fuzzing experience by guiding developer on determining their fuzzer's bottlenecks. It helps by providing
+aggregated and individual fuzzers reachability and coverage reports. Developer can either introduce a new 
+fuzz target or modify an existing one to improve the quality of the harness. To check some example cases 
```

Comment:
Maybe explain that improving a harness or introducing a new target can cover previously unreached code.